ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -435,7 +435,7 @@ private TaskTracker addLoadFunctionTasks(Context loadContext, BootstrapEventsIte
       // texts using new DB name. Currently it refers to the source database name.
     }
 
-    CreateViewDesc desc = new CreateViewDesc(dbDotView, table.getAllCols(), null, table.getParameters(),
+    CreateViewDesc desc = new CreateViewDesc(dbDotView, table.getCols(), null, table.getParameters(),
```

Comment:
Could you please describe the issue experienced, may be here or in JIRA? 
We don't need getPartCols during view replication?

Reply:
I have listed the steps to reproduce the issue in jira.
We do need getPartCols and its part of the constructor parameters for CreateViewDesc (its in next line so its not showing in the diff, expanded view will show). However list from getAllCols adds partition columns to both column list and partition column list and latter in the code flow the check fails due to duplicate entry.